### PR TITLE
Fix(GroupSystemMessages): add tests and adjust conditions for combined messages

### PR DESCRIPTION
**Pull request by DorraJaouad**
### ☑️ Resolves

Added tests are designated only for combined system message rendering

### 🚧 Tasks

- [ ] To add more tests for collapsed state (follow-up)

### 🏁 Checklist

- [ ] 🌏 Tested with Chrome, Firefox and Safari or should not be risky to browser differences
- [ ] 🖥️ Tested with Desktop client or should not be risky for it 
- [ ] 🖌️ Design was reviewed, approved or inspired by the design team
- [ ] ⛑️ Tests are included or not possible
- [ ] 📗 User documentation in https://github.com/nextcloud/documentation/tree/master/user_manual/talk has been updated or is not required

### CODE REVIEW

File: src/components/MessagesList/MessagesGroup/MessagesSystemGroup.spec.js [hunk 1]
```diff
@@ -0,0 +1,489 @@
...
+			previousMessageId: 90,
+			nextMessageId: 200,
+		})
+		expect(messagesEl.at(1).props()).toMatchObject({
+			message: {
+				id: MESSAGES[0].id,
+				message: MESSAGES[0].message,
+				actorId: MESSAGES[0].actorId,
+				actorDisplayName: MESSAGES[0].actorDisplayName,
+			},
```
Comment (Antreesy) [inline src/components/MessagesList/MessagesGroup/MessagesSystemGroup.spec.js:116]:
```suggestion
			message: MESSAGES[0],
```

**Review by Antreesy — COMMENTED** [review]
The rest looks good

**Review by Antreesy — APPROVED** [review]
Tested

File: src/components/MessagesList/MessagesGroup/MessagesSystemGroup.spec.js [hunk 2]
```diff
@@ -0,0 +1,479 @@
...
+	})
+
+	describe('renders grouped system message of user actions', () => {
+		let MESSAGES
+		/**
+		 * @param {object} wrapper - Wrapper of the component
+		 * @param {string} expectedMessage - Expected message
+		 *
+		 */
+		function testGroupedSystemMessages(wrapper, expectedMessage) {
```
[the remark this answers is not in the document]
Comment (DorraJaouad) [reply src/components/MessagesList/MessagesGroup/MessagesSystemGroup.spec.js:276]: you mean like different actors for adding users ? That's why I have added a third system message to show that they can't be grouped.

File: src/components/MessagesList/MessagesGroup/MessagesSystemGroup.spec.js [hunk 3]
```diff
@@ -0,0 +1,479 @@
...
+				systemMessage: 'user_removed',
+				timestamp: 200,
+				isReplyable: false,
+			}, {
+				id: 130,
+				token: TOKEN,
+				actorId: 'actor-3',
+				actorDisplayName: 'actor three',
+				actorType: ATTENDEE.ACTOR_TYPE.USERS,
+				message: 'Actor left the scene',
```
Comment (Antreesy) [inline src/components/MessagesList/MessagesGroup/MessagesSystemGroup.spec.js:366]: Then also replace a text here. I know it doesn't affect tests, but can confuse someone in the future

File: src/components/MessagesList/MessagesGroup/MessagesSystemGroup.spec.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Antreesy) [reply src/components/MessagesList/MessagesGroup/MessagesSystemGroup.spec.js:276]: Should be enough, thanks!

**Review by Antreesy — APPROVED** [review]

**Pull request merged**